www/README.md

```diff
@@ -18,18 +18,43 @@ npm start
 
 Edits made to react components in `src/` will be hot-reloaded in this doc site.
 
-## Developing with a theme applied
+## Including a custom theme
 
-1. Run
-```sh
-npm run develop:with-theme
-```
+1. Create a separate SASS stylesheet with your theme in `src/scss`, e.g. `src/scss/my-theme.scss`.
+2. Add your theme to `theme-config.js`, e.g.
```

Comment:
Maybe there is a reason to add a path to the `theme-config.js` to prevent confusions? Something like `Add your theme to `theme-config.js` (project_root/www/theme-config.js)`. What do you think?

Reply:
Hmm, I actually think it's fine right now, because this README is located in `www` already and I think it's clear that `theme-config.js` is located in the same directory

Reply:
Agreed, probably fine as is.